### DISCUSSION TITLE
百元快剪

### DISCUSSION THREAD

**Discussion by ALAN30359704** [1101-GS4719-C/Python-Community-GS4719-C#1460] [Q&A]
不知道中央的男女同學們，
對於百元快剪有什麼看法，
我本人覺得在我到山下剪一顆300塊的頭跟在學校附近100塊的頭其實差不多，
如果只是想剪短剃是不是百元快剪其實超賺？

**Comment by aa103967622** [comment]
我覺得要看你想修剪的髮型，設計師跟百元快剪的技術肯定有差，人家也不是在貴假的，如果你要的髮型本來就好剪，就像煮粥一樣，粥是可以煮得多難吃？我最近去一家500元的理髮，他們甚至會幫忙檢查你的頭皮情況，頭髮也相對百元快剪修得乾淨（百元快剪常常出現頭髮被剪得參差不齊的情況），這樣就覺得這錢花得值得多長。

**Comment by poweichen00** [comment]
我覺得剪頭髮看個人，確實跟樓上講得一樣剪的技術有差，如果你覺得感覺差不多那就可以選百元快剪，畢竟快又便宜。

**Comment by brian431** [comment]
沒錯，若是你想要的頭髮造型對於技術的要求不高，在細節上也沒有太多需要注意的，這時候高價位的剪髮就不會是你最佳的選擇。

**Comment by 1104030360** [comment]
我認為如果是沒有對頭髮造型要求非常高的話可以去剪百元快剪，如果想要求時尚流行還是要去髮廊，畢竟金錢換技術，但如果到時候當兵要剃頭的話我應該會找百元快剪

**Comment by roma0323** [comment]
只想剪短剃的話就隨便一家都可以喇，除非你有想法或是很注重細節再去300或以上的

**Comment by Alexgns** [comment]
如果對髮型沒有甚麼額外要求，當然選100元的就夠了，如果只有要剪短多付200元得到差不多的結果感覺有點虧，但也可試試看到較貴有設計師的地方嘗試看看變換造型

**Comment by alstinlin** [comment]
我覺得快剪跟幾百塊的還是有落差，不僅在造型上會粗糙一點，因為我自己頭髮比較容易亂翹，所以會希望理髮師幫我剪一個比較容易整理的髮型，通常去快剪就不太好意思這樣要求，不過百元快剪確實很快也很便宜。

**Comment by sumairu0727** [comment]
我曾經也認為我要的髮型很簡單應該去百元剪髮就可以搞定，直到有一次被某個快剪的搞到我就下定決心不會再去快剪店。
當然快剪店有時候可以撿到會剪的設計師，但就是要抱著可能會被剪爛的心情去

**Comment by Superlan8** [comment]
我小時候都是在百元快剪剪的，但上大學有燙頭髮後，我就不敢再百元快剪剪了，畢竟都花錢燙了，可不想再因為貪剪髮的便宜把燙的頭髮都剪壞，所以之後就都改在髮廊剪，固定設計師比較有保障。

**Comment by chia-kai-chang** [comment]
【大會報告】閒聊八卦而且沒有科技成分的文章，目前規劃不能得到點數。

**Comment by pudding0803** [comment]
或許是我沒有到特別在意頭髮（或只是沒什麼想法），所以我目前還沒有剪過一次超過 200 的🥹，不過改天也會想試試看，說不定有的會真的有特別之處！！

**Comment by Aaron0810** [comment]
我不行ㄝ，我從小到現在幾乎都給同一家理髮店同一個理髮師剪，甚至是會為了剪頭髮特地回家的那種(我家在台南)，只能說完全不相信那些百元快剪，而且也聽過不少被剪壞的頭髮，還是會怕，不過或許跟百元快剪也沒關係，就單純不會想給其他理髮師剪而已(可能算一種強迫症)

**Comment by Brownnnn78** [comment]
我在中壢都固定髮型師，每次要坐下就可以剪很好看，他都記得我想要的樣子，只要跟他聊天就好了，而且學生價才240雖然沒有洗頭但有沖水，還會幫你抓造型。真的建議各位社恐或怕剪壞的可以找到習慣的髮型師然後固定下來，頭髮是第二張臉啊！

**Comment by WeiYen1103** [comment]
我弟在國中前都是剪百元快剪，但我發現好像隨著年齡增長，不論是男是女，大家的審美提高，慢慢地就會比較少去那種理髮店剪髮了

**Comment by Boying12** [comment]
總的來說，如果需求只是簡單的剪髮，甚至是剃頭，百元快剪確實是一個非常划算的選擇，它能夠節省金錢和時間，並滿足基本的剪髮需求。然而，若是對髮型有個人化的需求，還甚高級髮廊的服務比較不會踩雷，不過相對錢錢的噴發是不可避免的。選擇的關鍵在於個人的需求與預期。
